{"name": "test_estimate_bridge_native_to_not_native[Ethereum (ETH)]", "status": "failed", "statusDetails": {"message": "AssertionError: Ожидался status = 200, пришел status = 400. Тело ответа: \n{\"ok\":false,\"data\":null,\"error\":\"0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee is incorrect token address(srcChainTokenIn)\"}.", "trace": "network = 'Ethereum (ETH)'\n\n    @pytest.mark.parametrize(\"network\", networks)\n    def test_estimate_bridge_native_to_not_native(network):\n        net = de_bridge_keys_data()[network].get('net')\n        native_token = de_bridge_keys_data()[network].get('nativeToken')\n        not_native_token = de_bridge_keys_data()[network].get('notNativeToken')\n>       DeBridge().EstimateBridge().estimate_bridge_native_to_not_native(net=net, native_token=native_token,\n                                                                         not_native_token=not_native_token)\n\ntests/api_tests/tests_de_bridge.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <serviceCalls.debrdige_service.DeBridge.EstimateBridge object at 0x10f3e1a60>\nnet = 'eth', native_token = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee'\nnot_native_token = '0x2a3bff78b79a009976eea096a51a948a3dc00e34'\n\n    def estimate_bridge_native_to_not_native(self, net, native_token, not_native_token):\n        response = requests.get(\n            f\"{self.url}?srcNet={net}&srcTokenAddress={native_token}\"\n            f\"&srcTokenAmount=1&dstNet={net}&dstTokenAddress={not_native_token}\")\n        logger.info(f\"Отправлен запрос на роут - {response.url}\")\n        logger.info(response.text)\n>       assert response.status_code == 200, f\"Ожидался status = {200}, \" \\\n                                            f\"пришел status = {response.status_code}. \" \\\n                                            f\"Тело ответа: \\n{response.text}.\"\nE       AssertionError: Ожидался status = 200, пришел status = 400. Тело ответа: \nE       {\"ok\":false,\"data\":null,\"error\":\"0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee is incorrect token address(srcChainTokenIn)\"}.\n\nserviceCalls/debrdige_service.py:39: AssertionError"}, "attachments": [{"name": "log", "source": "fbc740ed-70c7-4625-95f1-c696481e50f4-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "network", "value": "'Ethereum (ETH)'"}], "start": 1685615493877, "stop": 1685615494395, "uuid": "fa6954f9-9b17-418a-9682-f2efc92b9a14", "historyId": "379d3b8e7006db6d0b884f21181f6540", "testCaseId": "02aafaeb5599f02847fcd6b690f3a745", "fullName": "tests.api_tests.tests_de_bridge#test_estimate_bridge_native_to_not_native", "labels": [{"name": "parentSuite", "value": "tests.api_tests"}, {"name": "suite", "value": "tests_de_bridge"}, {"name": "host", "value": "MacBook-Air-Anna.local"}, {"name": "thread", "value": "18188-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_tests.tests_de_bridge"}]}